kafka_scanner/msg_processor_handlers.py

```diff
@@ -20,11 +20,23 @@ def retry_on_exception(exception):
 
 
 class MsgProcessorHandlers(object):
-    def __init__(self, encoding=None):
-        self.decompress_fun = zlib.decompress
+    def __init__(self, encoding=None, decompress=None, msgformat=None):
+        if decompress:
+            import zlib
```

Comment:
why lazy import of zlib? It is a standard library.

Reply:
sure, i can import it, there is no point in polluting the global namespace unnecesarily but it wouldnt be a big deal.